### DISCUSSION TITLE
Update `platform:gitea` label or add new `platform:forgejo` label

### DISCUSSION THREAD

**Discussion by HonkingGoose** [renovatebot/renovate#23446] [Suggest an Idea]
### Tell us more.

We have a `platform:gitea` label. As I understand things, Forgejo is compatible with Gitea?

I'll let you maintainers decide which is best:

1. Update `platform:gitea` to `platform:gitea-and-forgejo`
2. Create new label `platform:forgejo`

Remember to search for the `platform:gitea` text in the repository. There are some docs that refer to the "old" label that you'll have to update. 😉

**Comment by rarkins** [comment]
I think we should just update the label description

**Reply by HonkingGoose** [reply]
I'll let you update the label description then. 😉

**Comment by jamietanna** [comment]
We've now got two labels for this
